### PR TITLE
fix visual issue header on mobile

### LINKED ISSUE #179: Header logos are too big on mobile
Screenshot_20200407-101905_Chrome

I can take this issue since the bug was introduced by a PR created by me :( https://github.com/code4romania/ce-ma-fac/pull/177

### PR DESCRIPTION
**Pull request by danichim**
Closes #179 

Please mention the main changes this PR brings.
replace logos

### CODE REVIEW

**Comment by Utwo** [comment]
There was an issue already with moving the DSU logo inline with gov logo. #176  
This will revert that issue back :(

**Comment by aniri** [comment]
> There was an issue already with moving the DSU logo inline with gov logo. #176
> This will revert that issue back :(

@Utwo this will not revert those changes, this only resizes the logos a bit on small screens :) they are still displayed in one line as #176 said.

@danichim could you please also adjust the spacing between the logos? see https://stirioficiale.ro/informatii as an example :smile:

**Comment by aniri** [comment]
actually, there is a small change of plans. as @cipick made this fix directly in the reusable component https://github.com/code4romania/taskforce-fe-components/pull/148 . it's a better approach since the fix will also be applied in the other projects that use those components. @danichim @Utwo we should update this PR once the new components release is out :rocket:

**Comment by danichim** [comment]
when will be merged i will address it

**Comment by aniri** [comment]
> when will be merged i will address it

@danichim v1.0.10 of the components is out :rocket:

**Comment by cipick** [comment]
@danichim the issue is still reproducible on 320px
Screenshot 2020-04-10 at 11 37 22
anyway if you checkout develop and run `npm install` the issues should be fix :D

**Comment by danichim** [comment]
@aniri  i changed the package.json to use v1.0.10

File: frontend/src/App.scss [hunk 1]
```diff
@@ -13,15 +13,8 @@
   margin-bottom: 1.6rem;
 }
 
-.incubated-by-wrapper {
```
Comment (aniri) [inline frontend/src/App.scss:16]: this is still needed. please do not remove it.

**Pull request merged**